src/App.js

```diff
@@ -26,11 +26,9 @@ const App = () => {
           : ''}
       </div>
       {state.selectedStock
-        ? <>
-          <div>
+        ? <div>
```

Comment:
Good clean-up. Remember you can clean it even more if your render branch is "show this or show nothing":

```
{state.selectedStock && (
  <div>
    <Footer />
  </div>
)}
```

Reply:
ok, will do it